lib/dialogue-agent/conversation.ts

```diff
@@ -79,6 +80,7 @@ export interface ConversationDelegate {
 export interface ConversationState {
     history : Message[];
     dialogueState : Record<string, unknown>;
+    lastMessageId : number|null;
```

Comment:
Can this be ever null? Why? Let's make 0 or 1 the default.

Reply:
In that case should the default db value be 0 as well?

Reply:
It doesn't really matter, we'll always write a value to the row, but yeah it could be. (I don't remember if sqlite supports default clauses though)